arcade/draw_commands.py

```diff
@@ -125,13 +121,9 @@ def draw_arc_outline(center_x: float, center_y: float, width: float,
     if tilt_angle == 0:
         uncentered_point_list = unrotated_point_list
     else:
-        uncentered_point_list = []
-        for point in unrotated_point_list:
-            uncentered_point_list.append(rotate_point(point[0], point[1], 0, 0, tilt_angle))
+        uncentered_point_list = [rotate_point(point[0], point[1], 0, 0, tilt_angle) for point in unrotated_point_list]
 
-    point_list = []
-    for point in uncentered_point_list:
-        point_list.append((point[0] + center_x, point[1] + center_y))
+    point_list = [(point[0] + center_x, point[1] + center_y) for point in uncentered_point_list]
```

Comment:
This is probably faster

Reply:
It is faster indeed

Reply:
As I attempt to calibrate these kinds of changes, I appreciate the feedback! 😎